shared/tracker/index.js

```diff
@@ -43,61 +39,14 @@ export type TrackerProps = {
   isPrivate?: boolean
 }
 
-export function trackerPropsToRenderProps (props: TrackerProps): RenderPropsUnshaped {
-  const renderChangedTitle = props.trackerMessage
-  const failedProofsNotFollowingText = `Some of ${props.username}'s proofs couldn't be verified. Track the working proofs?`
-  const currentlyFollowing = !!props.lastTrack
-
-  const changed = !(props.proofs || []).every(function (proof, index, ar) {
-    return (!proof.meta || proof.meta === metaNone)
-  })
-
-  const reason = currentlyFollowing && renderChangedTitle ? renderChangedTitle : props.reason
-
+export function trackerPropsToRenderProps ({currentlyFollowing, inviteLink, isPrivate, lastAction,
```

Comment:
yes, cause we need to only return a subset of the keys due to $Shape being used. i originally tried to just remove this converter but flow complains